app/src/components/network/NetworkList.vue

```diff
@@ -40,23 +40,12 @@ export default {
   },
   computed: {
     ...mapGetters({ networkId: `network` }),
-    whichFlow() {
```

Comment:
this is not needed anymore? is this handled by the router? does the network list still work in the account creation flow?

Reply:
yes :shrug: everything works just fine. I mean, what is supposed to happen when you click on a NetworkItem? You are either taken to validators or to portfolio, right?

Reply:
we use the same component in the sign in flow. so please check if removing this breaks the sign in flow.

Reply:
oops, ok, will check it